Sourcery/Models/TypeName.swift

```diff
@@ -78,24 +79,22 @@ public final class TypeName: NSObject, AutoCoding, AutoEquatable, AutoDiffable,
                     unwrappedTypeName = name.drop(first: "ImplicitlyUnwrappedOptional<".characters.count, last: 1)
                 }
                 self.unwrappedTypeName = unwrappedTypeName.bracketsBalancing()
+                self.isGeneric = unwrappedTypeName.contains("<") && unwrappedTypeName.characters.last == ">"
```

Comment:
` 'self' captured by a closure before all members were initialized` is what happens then